Source/Particles/Collision/BinaryCollision/ProtonBoronFusionCrossSection.H

```diff
@@ -8,16 +8,100 @@
 #ifndef PROTON_BORON_FUSION_CROSS_SECTION_H
 #define PROTON_BORON_FUSION_CROSS_SECTION_H
 
+#include "Utils/WarpXConst.H"
+
 #include <AMReX_REAL.H>
 
+#include <cmath>
 
 /**
- * \brief Computes the total proton-boron fusion cross section. Needs to be implemented.
+ * \brief Computes the total proton-boron fusion cross section using the analytical fits given in
+ * W.M. Nevins and R. Swain, Nuclear Fusion, 40, 865 (2000). The result is returned in SI units
+ * (square meters).
+ * For the record, note that there is a typo in equation (1) of this paper: the total cross section
+ * should read S(E)/E*exp(-sqrt(E_G/G)) instead of S(E)/E*exp(sqrt(E_G/G)) (minus sign in the
+ * exponential).
+ *
+ * @param[in] E_kin_star the kinetic energy of the proton-boron pair in its center of mass frame,
+ * in SI units.
  */
 AMREX_GPU_HOST_DEVICE AMREX_INLINE
-amrex::ParticleReal ProtonBoronFusionCrossSection (const amrex::ParticleReal& /*E_star*/)
+amrex::ParticleReal ProtonBoronFusionCrossSection (const amrex::ParticleReal& E_kin_star)
 {
-    return amrex::ParticleReal(0.);
+    // If kinetic energy is 0, return a 0 cross section and avoid later division by 0.
+    if (E_kin_star == amrex::ParticleReal(0.)) {return amrex::ParticleReal(0.);}
+
+    // Fits use energy in keV and MeV
+    constexpr amrex::ParticleReal joule_to_kev = amrex::ParticleReal(1.e-3)/PhysConst::q_e;
+    constexpr amrex::ParticleReal joule_to_mev = amrex::ParticleReal(1.e-6)/PhysConst::q_e;
```

Comment:
Etc. for all `amrex::ParticleReal`
```suggestion
    constexpr amrex::ParticleReal joule_to_kev = 1.e-3_prt/PhysConst::q_e;
    constexpr amrex::ParticleReal joule_to_mev = 1.e-6_prt/PhysConst::q_e;
```